maven/plugins/org.jboss.tools.maven.springboot/src/org/jboss/tools/maven/springboot/configurators/SpringBootProjectConfigurator.java

```diff
@@ -210,6 +212,7 @@ private void configure(IMavenProjectFacade facade) {
 				public IStatus runInWorkspace(IProgressMonitor monitor) throws CoreException {
 					try {
 						MavenProject mavenProject = facade.getMavenProject(monitor);
+						ModuleCoreNature.addModuleCoreNatureIfNecessary(facade.getProject(), monitor);
```

Comment:
if we put ```ModuleCoreNature.addModuleCoreNature``` into ```#addUtilityFacet``` we'd have a single occurrence where module core is added (versus 2 right now). WDYT?

Reply:
Will do